project.clj

```diff
@@ -4,32 +4,32 @@
   :license {:name "MIT"
             :url  "https://opensource.org/licenses/MIT"}
 
-  :dependencies [[com.datomic/datomic-free "0.9.5206" :exclusions [joda-time]]
-                 [com.taoensso/timbre "4.3.1"]
+  :dependencies [
```

Comment:
All of these changes were to get rid of warnings from `lein deps :tree`. Nailing to a specific version of a dep usually helps, and removing unused stuff does as well.